src/Maker/MakeController.php

```diff
@@ -60,19 +62,34 @@ public function getParameters(InputInterface $input): array
         $controllerClassName = Str::asClassName($input->getArgument('controller-class'), 'Controller');
         Validator::validateClassName($controllerClassName);
 
+        $title = "Hello {$controllerClassName}!";
+        if (file_exists($this->projectDir.'/templates/base.html.twig')) {
+            $twigFirstLine = "{% extends 'base.html.twig' %}\n{% block title %}{$title}{% endblock %}";
+        } else {
+            $twigFirstLine = "<!DOCTYPE html>\n<title>{$title}</title>";
+        }
+
         return [
             'controller_class_name' => $controllerClassName,
             'route_path' => Str::asRoutePath(str_replace('Controller', '', $controllerClassName)),
             'route_name' => Str::asRouteName(str_replace('Controller', '', $controllerClassName)),
+            'twig_file' => Str::asFilePath(str_replace('Controller', '', $controllerClassName)).'.html.twig',
+            'twig_first_line' => $twigFirstLine,
         ];
     }
 
     public function getFiles(array $params): array
     {
-        $skeletonFile = $this->isTwigInstalled() ? 'ControllerWithTwig.tpl.php' : 'Controller.tpl.php';
```

Comment:
Now that we have php support in the templates, could we unify these 2 files into 1? Unless they are VERY different, that seems less error-prone and the files will stay in sync with each other.